Specs/Scene/PickSpec.js

```diff
@@ -94,6 +110,35 @@ defineSuite([
         expect(scene).toPickPrimitive(rectangle);
     });
 
+    it('picks a primitive at pickSize extent', function() {
```

Comment:
`pickSize` isn't a name used by the API. Maybe instead `picks a primitive with a given width and height`.